rust/src/tests/mod.rs

```diff
@@ -1,11 +1,7 @@
 use std::os::unix::fs::MetadataExt as StdMetadataExt;
 
-use std::{
-    fs::metadata,
-    path::Path,
-};
+use std::{fs::metadata, path::Path};
```

Comment:
Again, there are lots of formatting changes here.  You shouldn't combine those with content changes.

Reply:
Sorry again, definitely a bad habit that I should be more careful of...